app/api/src/controllers/taskController.ts

```diff
@@ -112,3 +112,173 @@ export const updateTaskHandler = asyncHandler(
     });
   },
 );
+
+// @desc Creates new planned task list for next day
+// @route POST /task/plan
+// @access Private
+export const planTaskHandler = async (
+  req: AuthRequest,
+  res: Response,
+): Promise<void> => {
+  try {
+    const date = req.query.date;
+
+    if (!date) {
+      res.status(422).json({ error: 'Missing date in query' });
+      return;
+    }
+
+    // Retrieves tasks from prior day
+    const from = moment(date as string)
+      .subtract(1, 'days')
+      .startOf('day');
+    const to = moment(from).add(1, 'day');
+
+    const today = moment(date as string)
```

Comment:
It should already be a string here right? Because !date checks that its not null/undefined